<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="11" column="1">
     <widget class="QLabel" name="odd">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QTableWidget" name="tableD"/>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="remove">
      <property name="minimum">
       <number>-9999</number>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QPushButton" name="sort_C">
      <property name="text">
       <string>Sort </string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QPushButton" name="fillC">
      <property name="text">
       <string>Fill C</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="add"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Erase n elements starting from</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="2">
     <widget class="QPushButton" name="merge">
      <property name="text">
       <string>Merge A and C</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="set_A">
      <property name="text">
       <string>SetA</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>n</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spin_A"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="sort_A_2">
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spin_B"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="set_B">
      <property name="text">
       <string>SetB</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Find first odd in A</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QTableWidget" name="tableC"/>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="from"/>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="n_n"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QTableWidget" name="tableB"/>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QPushButton" name="insert">
      <property name="text">
       <string>Delete n elements from and insert B into A</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sort_A">
      <property name="text">
       <string>Sort rev</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTableWidget" name="tableA"/>
    </item>
    <item row="17" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Count odd in merget</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLabel" name="odd_in_merget">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
